src/index.js

```diff
@@ -70,116 +73,152 @@ query ($id: ID!) {
+      }
+    }
   });
-  console.log(resp.status);
-  console.log(resp.data);
+
+  return members;
 }
 
-function check() {
+/**
+ * Request the GraphQL API of Github with SPRINTS_BACKLOG_CARDS_QUERY query
+ *
+ * @param {Array} members - array of object contains members list
+ * @returns {Array} - returns the parsed output of query.
+ */
+function backlogCardQuery(members) {
   return octokit
-    .graphql(CARDS_QUERY, { id: COLUMN_NODE_ID })
+    .graphql(SPRINTS_BACKLOG_CARDS_QUERY, { id: COLUMN_NODE_ID })
     .then((query) => {
-      console.log(query);
-      console.log(query.node.cards.nodes);
-      console.info(
-        "Preview:\n" + prsToText(extractPullRequests(query.node.cards.nodes))
-      );
-    })
-    .catch((err) => {
-      console.error("Can't make test request\n", err);
-      process.exit(1);
+      return parseQuery(members, query.node.cards.nodes);
     });
 }
 
-const SOZVON_MSG = `☝️
-Join the meeting in Discord!
-@specc @guryn @khaydarovm @nikmel2803 @gohabereg @ilyamore88 @GeekaN @augustovich @n0str @f0m41h4u7 @polina_shneider @oybekmuslimov @xemk4`
+/**
+ * Provides list of members with there task
+ *
+ * @param {string} memberList - contains memberList with space as separator
+ * @returns {Array} - returns Array of object contains user name and it's task
+ */
+const getMembersName = (memberList) => {
+  const members = [];
+
+  if (memberList) {
+    memberList.split(' ').forEach((items) => {
+      members.push({
+        name: items,
+        tasks: [],
+      });
+    });
+
+    return members;
+  }
+
+  return octokit
+    .graphql(MEMBERS_QUERY)
+    .then((query) => {
+      query.organization.membersWithRole.nodes.forEach((items) => {
+        members.push({
+          name: items.login,
+          tasks: [],
+        });
+      });
+
+      return members;
+    });
+};
+
+/**
+ * Use to create message for telegram bot.
+ * It parse the Array of object with members and it's task into message.
+ *
+ * @returns {string} - Parsed message for telegram bot
+ */
+async function notifySprintsBacklogs() {
+  let dataToSend = "📌 Sprint's backlog \n\n";
+  const response = await backlogCardQuery(await getMembersName(MENTION));
 
+  response.forEach((items) => {
+    dataToSend += (`@${items.name}`);
+    dataToSend += '\n';
+    items.tasks.forEach((data) => {
+      dataToSend += (`⚡️ ${data} \n`);
+    });
+    dataToSend += '\n\n';
+  });
+
+  return dataToSend;
+}
+
+/**
+ * Call the Github GraphQL API, parse its response to message and add that message as cron job.
+ */
 async function main() {
-  await check();
-  console.log("Successful check, continuing");
   const job = new CronJob(
     PR_TIME,
-    () => {
-      console.log("Firing pr job");
-      sendOpenPrs()
-        .then(() => console.log("Job completed"))
+    async () => {
+      notify(await notifySprintsBacklogs())
+        .then(() => console.log('Job completed'))
         .catch(console.error);
     },
```

Comment:
Maybe you should use async\await everywhere instead if then\catch

Reply:
then I need to use try/catch also to catch the error.
It is important to change from then\catch to async\await?